app/lib/publisher/backend.dart

```diff
@@ -169,6 +206,41 @@ class PublisherBackend {
     });
   }
 
+  /// A callback from consent backend, when a consent is granted.
+  Future inviteConsentGranted(String publisherId, String userId) async {
```

Comment:
Couldn't we wait with creating the `PublisherMember` until we get here? It's up the consent backend to do deduplication and avoid sending emails too frequently... Maybe the consent backend should offer a flow to query for pending consent flows... hmm, maybe... I'm not sure... it just seems we're duplicating data, and we don't really want the `isPending` in `PublisherMember`..